presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java

```diff
@@ -68,6 +84,19 @@ private static TableDefinition partitionedTableDefinition()
                 .build();
     }
 
+    private static TableDefinition partitionedTableWithVariablePartitionsDefinition()
+    {
+        String tableName = "partitioned_table_with_too_many_partitions";
+        String createTableDdl = "CREATE TABLE %NAME%(col INT) " +
+                "PARTITIONED BY (part_col INT) " +
```

Comment:
it would be nice to partition this table by two columns

Reply:
There is a test covering that: `TestHiveIntegrationSmokeTest#insertPartitionedTable()`